complex/ft_endpoint.c

```diff
@@ -43,10 +43,6 @@ int ft_open_passive(void)
 	if (pep)
 		return 0;
 
-	ret = ft_open_control();
```

Comment:
<code>ft_open_active</code> is actually called from another place, where we need <code>ft_open_control</code> call. So only removed it from <code>ft_open_passive</code>.